src/FixedDataTable.js

```diff
@@ -384,6 +404,7 @@ class FixedDataTable extends React.Component {
 
   static defaultProps = /*object*/ {
     elementHeights: {
+      cellGroupWrapperHeight: 0,
```

Comment:
Would defaulting this to undefined be technically more correct?

Reply:
Yes, that makes more sense as we use '||' to test cellGroupWrapperHeight at different parts. Will change.